backend/frontend/templates/frontend/user/component_card.html

```diff
@@ -1,25 +1,144 @@
+        <div class="flex gap-2 items-center flex-wrap">
+            {% for tag_name in card.tag_names %}
+                <span class="flex px-2 py-1 rounded-full border border-gray-300 text-xs">{{tag_name}}</span> 
+            {% endfor %}
+        </div>
+    {% endif %}
 
-    {% if card.request_user_can_start %}
+    {% if card.due_time %}
+        <div class="flex gap-1" title="Due time">
+            <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill="currentColor" d="M12 20a7 7 0 0 1-7-7a7 7 0 0 1 7-7a7 7 0 0 1 7 7a7 7 0 0 1-7 7m0-16a9 9 0 0 0-9 9a9 9 0 0 0 9 9a9 9 0 0 0 9-9a9 9 0 0 0-9-9m.5 4H11v6l4.75 2.85l.75-1.23l-4-2.37V8M7.88 3.39L6.6 1.86L2 5.71l1.29 1.53l4.59-3.85M22 5.72l-4.6-3.86l-1.29 1.53l4.6 3.86L22 5.72Z"/></svg>
+            {% now "Y-m-d H:i:s" as current_datetime %}
+            {% if card.due_time|date:"U" > current_datetime|date:"U" %}
+                <p class="text-red-600">
+                    {{card.due_time|naturalday}} 
+                </p>
+                {% else %}
+                <p>
+                    {{card.due_time|naturalday}} 
+                </p>
+            {% endif %}
+        </div>
+    {% endif %}
+    
+    {% if card.request_user_is_assignee == False %}
+        <div class="w-full" >
+            <p class="{{styles.small_bold_text}}">Assignees</p>
+            <p class="truncate" title="{{card.assignee_names.0}}">{{card.assignee_names.0}}</p> 
+        </div>
+    {% endif %}
 
-    <button type="button" class="{{ styles.button_primary_small }}"
-        hx-post="{% url 'action_start_card' card.id %}"
-        hx-target="#card_{{ card.id }}"
-        hx-swap="outerHTML"
-        >Start
-    </button>
+    {% if card.reviewer_names %}
+        <div class="w-full">
+            <p class="{{styles.small_bold_text}}">Reviewers</p>
+            {% for reviewer_name in card.reviewer_names %}
+                <p class="truncate" title="{{reviewer_name}}">{{reviewer_name}}</p> 
+            {% endfor %}
+        </div>
+    {% endif %}
 
+    {% if card.repo_url %}
+        <div class="flex gap-1">
+            <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill="currentColor" d="M12 2A10 10 0 0 0 2 12c0 4.42 2.87 8.17 6.84 9.5c.5.08.66-.23.66-.5v-1.69c-2.77.6-3.36-1.34-3.36-1.34c-.46-1.16-1.11-1.47-1.11-1.47c-.91-.62.07-.6.07-.6c1 .07 1.53 1.03 1.53 1.03c.87 1.52 2.34 1.07 2.91.83c.09-.65.35-1.09.63-1.34c-2.22-.25-4.55-1.11-4.55-4.92c0-1.11.38-2 1.03-2.71c-.1-.25-.45-1.29.1-2.64c0 0 .84-.27 2.75 1.02c.79-.22 1.65-.33 2.5-.33c.85 0 1.71.11 2.5.33c1.91-1.29 2.75-1.02 2.75-1.02c.55 1.35.2 2.39.1 2.64c.65.71 1.03 1.6 1.03 2.71c0 3.82-2.34 4.66-4.57 4.91c.36.31.69.92.69 1.85V21c0 .27.16.59.67.5C19.14 20.16 22 16.42 22 12A10 10 0 0 0 12 2Z"/></svg>
+            <a class="{{styles.link}}" href="{{card.recruit_project.repository.full_name|convert_git_url:card.recruit_project.repository.owner}}">Repository</a>  
```

Comment:
cant we just use the owner/repo-name here

Reply:
My thinking behind this was that the repo name didn't really provide any useful info and that it was too long and hard to present visually without cutting it. I may be wrong, I've never needed to know or see th repo name

Reply:
I meant on extracting the https:// version of the repo. We have owner and repo name as fields on each Repository instance.

Reply:
The owner name is stored is currently `Umuzi.org` but the ones for valid links is `Umuzi-org` so I couldn't find any easy way to replace the dot with a dash to make the links valid :grimacing: